News/Main.py

```diff
@@ -29,8 +29,9 @@ def App():
         if newsSourceNumber==99:
             sys.exit()
         if (newsSourceNumber >=len(newsSources)):
-            print ("Please select the index no less than "+ str(len(newsSources)))  
-        obj=NewsPulling(newsSources[newsSourceNumber])
+            print ("Please select the index no less than "+ str(len(newsSources)))

```

Comment:
You need to add some info to let the users know that we also provide the cricket scores. Similar to the news.?

Reply:
Its already there: https://github.com/Griffintaur/News-At-Command-Line/pull/41/files#diff-0f1be8eec416e71c28b41efb0544df26R25

Also have a look at the updated screenshots in one of the binary files:
https://github.com/Griffintaur/News-At-Command-Line/pull/41/commits/555c97aa11bf873e704b84297e7916be8ad96f0c

Reply:
![image](https://user-images.githubusercontent.com/2469198/48529204-ed379e00-e846-11e8-8a2d-5c09ad6f6482.png)

Reply:
Great @adilansari